src/main/java/org/ow2/proactive/procci/model/occi/metamodel/Mixin.java

```diff
@@ -118,6 +116,10 @@ public MixinRendering getRendering() {
                         .stream()
```

Comment:
The code to map a Category to its title and returns a List is duplicated 3 times:

```
.depends(this.actions
                     .stream()
                      .map(depend -> depend.getTitle())
                      .collect(Collectors.toList()))
```

This could be prevented as follows:

` .actions(mapTitle(actions))`

```
private List<String> mapTitle(List<? extends Category> input) {
    return input.stream()
            .map(action -> action.getTitle())
            .collect(Collectors.toList());
}
```